mvc/migration/migratingfrommvc5.rst

```diff
@@ -1,257 +1,244 @@
+The completed sample (this tutorial and the next two) will demonstrate how to migrate an MVC 5 web project that includes controllers, views, ASP.NET Identity models, startup and configuration code found in many MVC 5 projects.
 
-Create the Destination Solution
--------------------------------
+*Optional:* Change the name of the Solution from *WebApp1* to *Mvc5*. Visual Studio will display the new solution name (*Mvc5*), which will make it easier to tell this project from the next project. 
 
-We will begin our migration by creating a new, empty ASP.NET 5 solution. Create a new project in Visual Studio 2015, choose an ASP.NET Web Application, and then choose the ASP.NET 5 Empty template.
+Create the MVC 6 project
+========================================
 
-.. image:: migratingfrommvc5/_static/new-project-mvc6.png
+Create a new empty MVC 6 web app with the same name as the previous project (*WebApp1*) so the namespaces in the two projects match. Having the same namespace makes it easier to copy code between the two projects. You'll have to create this project in a different directory than the previous project to use the same name.
 
 .. image:: migratingfrommvc5/_static/new-project-select-empty-aspnet5-template.png
 
-This migration will start from an empty template. If you're already familiar with ASP.NET 5 and its starter templates and there are features in a starter template you would like to take advantage of, you may wish to start from another template. The next step is to configure the site to use MVC. This requires changes to the project.json file and Startup.cs file. First, open project.json and add "Microsoft.AspNet.Mvc" to the "dependencies" property:
+- *Optional:* Create a new MVC 6 app named *WebApp1* with the full template. You can look at the template generated code to see the end result or to copy code to the conversion project. It's also helpful when you get stuck on a conversion step to compare with the template generated project.
+
+Configure the site to use MVC
+========================================
+
+- Open the *project.json* file and add ``Microsoft.AspNet.Mvc`` to the ``dependencies`` property and the scripts section as highlighted below:
+
+.. literalinclude:: migratingfrommvc5/samples/WebApp1/src/WebApp1/project.json
+	:language: json
+	:emphasize-lines: 8-9, 29-31
+  
+``Microsoft.AspNet.StaticFiles`` is the static file handler. The ASP.NET runtime is modular, and you must explicitly opt in to serve static files.
 
-.. code-block:: javascript
+The ``scripts`` section is used to specify when specified build automation scripts should run. Visual Studio now has built-in support for running scripts before and after specific events. The ``scripts`` section above specifies `NPM <https://docs.npmjs.com/>`__, `Bower <http://bower.io/>`__ and `Gulp <http://gulpjs.com/>`__ scripts should run on the ``prepublish`` stage.  We'll talk about NPM, Bower, and Gulp later in the tutorial.
```

Comment:
Seems you should explain more about prepublish or point to more information about how this is done or the available options. Tough to see without the above json.